settings.gradle

```diff
@@ -1,4 +1,4 @@
 rootProject.name = 'terra-billing-profile-manager'
 include('service', 'client')
 
-gradle.ext.releaseVersion = '0.1.566-SNAPSHOT'
+gradle.ext.releaseVersion = '0.1.567-SNAPSHOT'
```

Comment:
Doesn't this bump automatically?

Reply:
I think that was from rebasing master - I didn't set it manually.

Reply:
Huh, I'm just confused about why the diff is showing any change then as merging a PR to main shouldn't change this